server/paymentProviders/stripe/virtual-cards.js

```diff
@@ -201,6 +220,8 @@ export const processAuthorization = async (stripeAuthorization, stripeEvent) =>
       CreatedByUserId: UserId,
       amount,
     });
+
+    await sendPurchaseNotifyEmail(virtualCard, amount, currency);
```

Comment:
If you go with the Activity solution, you would just need to create a VIRTUAL_CARD_PURCHASE activity here with the necessary data.

Reply:
@kewitz : Done. Let me know if all looks good now. 👍🏽